routemaster/client/errors.rb

```diff
@@ -0,0 +1,11 @@
+module Routemaster
+  class Client
+    class Error < StandardError; end
+
+    class InvalidArgumentError       < Error; end
```

Comment:
yeah we can do that, I just liked the idea of all the errors stemming from a base `Routemaster::Client::Error` class, but am happy to change it.